graph/mongo/quadstore.go

```diff
@@ -113,38 +114,57 @@ func ensureIndexes(db *mgo.Database) error {
 }
 
 func createNewMongoGraph(addr string, options graph.Options) error {
-	conn, err := mgo.Dial(addr)
+	conn, err := dialMongo(addr, options)
 	if err != nil {
 		return err
 	}
 	defer conn.Close()
 	conn.SetSafe(&mgo.Safe{})
+	db := conn.DB("")
```

Comment:
Yes, that will just use the DB specified in DialInfo (i.e., the one specified as default or the one explicitly specified in the options).